plugins/commando.c

```diff
@@ -1051,14 +1135,24 @@ static struct command_result *listdatastore_done(struct command *cmd,
 
 	json_array_start(js, "runes");
 	json_for_each_arr(i, t, d) {
+		const struct rune *this_rune;
 		const jsmntok_t *s = json_get_member(buf, t, "string");
 		if (runestr != NULL && !json_tok_streq(buf, s, runestr)) 
 			continue;
-		json_add_runestr(js, buf + s->start, s->end - s->start, true);
+		if(rune) {
+			this_rune = rune;
+		} else {
+			this_rune = rune_from_base64n(tmpctx, buf + s->start, s->end - s->start);
+			if(this_rune == NULL) {
+				plugin_log(plugin, LOG_BROKEN, "Invalid rune %.*s", s->end - s->start, buf + s->start);
```

Comment:
Maybe a comment here?  /* We should not have an invalid rune in the datastore! */